utilities/plotting/useUpdatePlot.m

```diff
@@ -0,0 +1,16 @@
+% A simple function to make an updating plot...
+
+% Make the figure
+h = figure(1000);
```

Comment:
@arwelHughes This design means the user can only have a single live plot, if they wanted to keep an old plot an run again they cannot this will overwrite the old one. Is this intended?